inc/jeo-scripts/L.TileLayer.BetterWMS.js

```diff
@@ -92,60 +93,92 @@ L.tileLayer.betterWms = function (url, options) {
 };
 
 //added H.E
-function buildpopup(content){
-    var record; var full_name_of_filename;
+function buildpopup(content, info_title, info_attributes, info_detail){
+  if( ((info_title !== null) && (info_title != "")) || ((info_attributes !== null) && (info_attributes != "")) || ((info_detail !== null) && (info_detail != ""))){
+    var record;
+    var regexp = /(https?:\/\/([-\w\.]+)+(:\d+)?(\/([-\w\/_\.]*(\?\S+)?)?)?)/ig;
+    var translations = {
+      en:  { view_detail: "View detail"},
+      km:  { view_detail: "មើលលម្អិត" },
+      my:  { view_detail: "ကြည့်ရန်အသေးစိတ်" },
+      lo:  { view_detail: "ເບິ່ງ​ລາຍ​ລະ​ອຽດ" },
+      th:  { view_detail: "ดูรายละเอียด" },
+      vi:  { view_detail: "Xem chi tiết" }
+    };
+
+    var view_detail_label = translations.en.view_detail;
+    if(detect_lang_site != "en-US"){
```

Comment:
@Huyeng Where is the **detect_lang_site** variable declared? I do not see it being declared/initialized on this function